src/openrct2/ride/Ride.cpp

```diff
@@ -1409,6 +1412,8 @@ void ride_construction_set_default_next_piece()
     if (ride == nullptr)
         return;
 
+    const auto& rtd = ride->GetRideTypeDescriptor();
```

Comment:
No need for `&` if it's a pointer (applies to multiple other places).

You might not need to change this based on my other comment's outcome.